crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -4584,6 +4626,11 @@ You may have it installed on another user account, but it is not available for t
     .with_clipboard(webview_attributes.clipboard)
     .with_hotkeys_zoom(webview_attributes.zoom_hotkeys_enabled);
 
+  #[cfg(windows)]
+  if let Some(environment) = &web_context.environment {
+    webview_builder = webview_builder.with_environment(environment.clone());
```

Comment:
The whole environment thing is a bit confusing now, that we have it in `on_new_window` and tell people to pass it in with `with_environment`/`window_features`, but we also have it done automatically here

Some documentation around the related attributes would be nice

Reply:
well i'm not sure if we should be explicit or not. like only validate the environment options matches? though in this case we get to that "error prune" validation issue..

since the on_new_window environment is all handled by window_features i think it's ok to keep it as is, so it's explicitly matched instead of relying on internal details

but i agree documentation should be improved anyway

Reply:
> well i'm not sure if we should be explicit or not. like only validate the environment options matches? though in this case we get to that "error prune" validation issue..

I think forcing the same environment makes sense here, no matter the options, it's too error prune the other way around

> since the on_new_window environment is all handled by window_features i think it's ok to keep it as is, so it's explicitly matched instead of relying on internal details

If we want to force the environment, keeping it feels like some extra confusions, anyways, we can't remove it in v2 anyways, let's just update documentation